recipes/uropa/meta.yaml

```diff
@@ -8,10 +8,10 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   fn: {{ name }}-{{ version }}.tar.gz
-  md5: 10e53bad46946c6c5f29a577de1220a7
+  md5: b211485b634763affdc7a51c1035426c
 
 build:
-  skip: True # [not py27]
+  skip: True # [not py34]
```

Comment:
Bioconda is currently building for py27, py35 and py36 so  if you want it to be build for only python 3 versions you should change it to `skip: True  # [py27]

Reply:
That was why the travis skipped the recipe.. thanks for pointing this out!